test/test/AppDelegate.swift

```diff
@@ -13,8 +13,8 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
 
-
-    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey: Any]?) -> Bool {
+    @nonobjc
```

Comment:
What warning? It's funny that we need to tell Travis the file is Obj-C and then also later that it isn't.